src/features/DisplayControls/Treatments/tests/TreatmentsUtils.spec.jsx

```diff
@@ -78,4 +82,42 @@ describe("TreatmentsUtils", () => {
       });
     });
   });
+
+  it("should return drug name with strike-through when dateStopped is present and instructions are not assigned", () => {
+    const drugOrderObject = {
+      drugOrder: {
+        drug: { name: "Paracetamol" },
+        dateStopped: "2023-01-01",
+      },
+      instructions: null,
+      additionalInstructions: null,
+    };
+
+    const { queryByTestId, queryByText } = render(getDrugName(drugOrderObject));
+
+    expect(queryByText("Paracetamol")).toBeTruthy();
+    expect(
+      queryByText("Paracetamol").classList.contains("strike-through")
+    ).toBeTruthy();
+    expect(queryByTestId("notes-icon")).toBeFalsy();
+  });
+
+  it("should return drug name with strike-through when dateStopped is null and instructions are not assigned", () => {
```

Comment:
Is this test name correct? or are we expecting something else?

Reply:
Thanks Arjun, It was a mistake 

It should be without strike-through, I'll fix and push the change